charts/sentry/values.yaml

```diff
@@ -1753,7 +1753,7 @@ ingress:
   # if you are using AWS ALB Ingress controller, switch this to 'aws-alb'
   # if you are using GKE Ingress controller, switch this to 'gke'
   regexPathStyle: nginx
-  # ingressClassName: nginx
+  ingressClassName: nginx  # traefik, istio, haproxy and other
```

Comment:
I'm fine with keeping the comments but I'd prefer users to purposely make to choice on which ingress controller to choose.